Blockzilla/OverlayView.swift

```diff
@@ -15,19 +15,40 @@ protocol OverlayViewDelegate: class {
 
 class OverlayView: UIView {
     weak var delegate: OverlayViewDelegate?
-    private var searchButtonGroup = [InsetButton]()
-    private let searchSuggestionsCount = 4 // Five search buttons in total since indexing starts from 0.
-    private var presented = false
-    private var searchQuery = ""
     private let copyButton = UIButton()
     private let findInPageButton = InsetButton()
+    private let searchSuggestionsPrompt = SearchSuggestionsPromptView()
+    private let searchButton = InsetButton()
+    private var searchButtonGroup = [InsetButton]()
+    private let searchSuggestionsCount = 4 // Five search buttons in total since indexing starts from 0.
     private let topBorder = UIView()
     public var currentURL = ""
-
+    private var presented = false
+    private var searchQuery = ""
+    
     init() {
         super.init(frame: CGRect.zero)
         KeyboardHelper.defaultHelper.addDelegate(delegate: self)
         
+        searchSuggestionsPrompt.backgroundColor = UIConstants.colors.background
+        searchSuggestionsPrompt.clipsToBounds = true
+        addSubview(searchSuggestionsPrompt)
+        
+        searchSuggestionsPrompt.snp.makeConstraints { make in
+            make.top.leading.trailing.equalTo(safeAreaLayoutGuide)
+        }
+        
+        topBorder.isHidden = true
```

Comment:
See above: do all of these lines need to be moved for proper functionality / logic?